docs/en/loading/load_concept/strict_mode.md

```diff
@@ -141,8 +141,11 @@ The preceding code snippet uses HDFS as an example. For detailed syntax and para
 ### INSERT
 
 ```SQL
-SET enable_insert_strict = {true | false};
-INSERT INTO <table_name> ...
+INSERT INTO [<database_name>.]<table_name>
+PROPERTIES(
+    "strict_mode" = "{true | false}"
```

Comment:
Except Stream Load, for all other loading methods, `strict_mode` is set the same way in PROPERTIES clause.

Reply:
done